spark/src/main/scala/geotrellis/spark/rasterize/RasterizeRDD.scala

```diff
@@ -0,0 +1,97 @@
+package geotrellis.spark.rasterize
+
+import geotrellis.vector._
+import geotrellis.raster._
+import geotrellis.spark._
+import geotrellis.spark.tiling._
+import geotrellis.raster.rasterize._
+import geotrellis.raster.rasterize.Rasterizer.Options
+import org.apache.spark.{HashPartitioner, Partitioner}
+import org.apache.spark.rdd._
+import scala.collection.immutable.VectorBuilder
+
+object RasterizeRDD {
```

Comment:
Do you think it'd be worth it to have another overload method or two that doesn't require `options`?

```scala
def fromGeometry[G <: Geometry, T: Numeric](
    geoms: RDD[G],
    layout: LayoutDefinition,
    ct: CellType,
    value: T
  ): RDD[(SpatialKey, Tile)] with Metadata[LayoutDefinition] = {
    fromGeometry(geoms, layout, ct, value, new HashPartitioner(geoms.sparkContext.defaultParallelism), Options.DEFAULT)
  }

  def fromGeometry[G <: Geometry, T: Numeric](
    geoms: RDD[G],
    layout: LayoutDefinition,
    ct: CellType,
    value: T,
    numPartitions: Int
  ): RDD[(SpatialKey, Tile)] with Metadata[LayoutDefinition] = {
    fromGeometry(geoms, layout, ct, value, new HashPartitioner(numPartitions), Options.DEFAULT)
  }
```